client/lang/src/en.json

```diff
@@ -7,7 +7,7 @@
     "Admin.BATCH_UNPUBLISH_PROMPT": "You have {num} page(s) selected.\n\nDo you really want to unpublish",
     "Admin.SELECTONEPAGE": "Please select at least one page",
     "Admin.FormatExample": "Example: {format}",
-    "Admin.NO_SIZE": "N\/A",
+    "Admin.NO_SIZE": "N/A",
```

Comment:
Is this correct or is it a serialisation/deserialisation error?

Reply:
I think it's correct and it was wrong before, in the .js file (which is what the .json file is turned int) - it's "N/A" - https://github.com/silverstripe/silverstripe-admin/blob/1/client/lang/en.js#L17